dynacred/src/Storage.ts

```diff
@@ -32,13 +32,22 @@ export class StorageDB {
     }
 
     static async getObject(entry: string): Promise<any> {
```

Comment:
A common test of the return value is `Object.keys(got).length > 0`, which is to check for the absence of relevant value. Using `undefined` explicit the missing object.
 
```suggestion
     static async getObject(entry: string): Promise<any | undefined> {
```